### PR TITLE
PHP 5.5.9, Ubuntu 14.04, Symfony 2.3.33, PHPUnit 4.8.13, install scripts

### PR DESCRIPTION
**Pull request by fpapadopou**
### ChangeLog
- Modified install scripts and made them compatible to Ubuntu 14.04.
- Made Apache install scripts compatible with 2.4.7
- Moved code & unit/functional tests to PHP 5.5.9
- Updated PHPUnit to 4.8.13, updated its deps too
- Updated Symfony to version 2.3.33 (current 2.3 latest)
### Merge process
- Will need to change the `library` parameter into `library_manager` (for some reason, this is on master branch but the parameter on production is still named `library`)
- Will need composer install after update

### CODE REVIEW

**Comment by fpapadopou** [comment]
@dastergon please take a look at the install scripts (Symfony & Apache), and if everything looks good, please assign @spiliot

File: scripts/install.sh [hunk 1]
```diff
@@ -50,26 +50,11 @@ rm -rf Symfony/app/logs/*
...
-		echo "`pwd`/logs-fs `pwd`/Symfony/app/logs/ ext4 loop,acl 0 0" | sudo tee -a /etc/fstab > /dev/null 2>&1
-		cat /etc/fstab
-
-		sudo mount `pwd`/Symfony/app/cache/
-		sudo mount `pwd`/Symfony/app/logs/
-
-		sudo rm -rf `pwd`/Symfony/app/cache/*
-		sudo rm -rf `pwd`/Symfony/app/logs/*
-
+        # Set access control for both apache and current user on cache and logs directories
```
Comment (dastergon) [inline scripts/install.sh]: Needs a fix in the indentation(check spaces/tabs) inside the condition

File: scripts/install.sh [hunk 2]
```diff
@@ -50,26 +50,11 @@ rm -rf Symfony/app/logs/*
...
-
-		sudo dd if=/dev/zero of=cache-fs bs=1024 count=0 seek=200000
-		sudo dd if=/dev/zero of=logs-fs bs=1024 count=0 seek=200000
-
-		yes | sudo mkfs.ext4 cache-fs
-		yes | sudo mkfs.ext4 logs-fs
-
-		mkdir -p `pwd`/Symfony/app/cache/
+	    # Need to create cache and logs directories, as they do not pre-exist in new deployments
+	    mkdir -p `pwd`/Symfony/app/cache/
```
Comment (dastergon) [inline scripts/install.sh]: Needs a fix in the indentation(check spaces/tabs) inside the condition

**Comment by fpapadopou** [comment]
@spiliot since you've already checked the compiler & eratosthenes services, could you please take a look at this PR too?

File: scripts/install.sh (hunk 1 above)
Comment (fpapadopou) [inline scripts/install.sh]: Done

File: scripts/install.sh (hunk 2 above)
Comment (fpapadopou) [inline scripts/install.sh]: Done

File: scripts/apache_install.sh [hunk 3]
```diff
@@ -4,8 +4,12 @@ set -e
 
 PACKAGENAME=builder
 
+<<<<<<< HEAD
```
Comment (dastergon) [inline scripts/apache_install.sh]: Conflict markers left inside the script

File: scripts/apache_install.sh [hunk 4]
```diff
@@ -4,8 +4,12 @@ set -e
...
 sudo ln -s /opt/codebender/$PACKAGENAME/Symfony/web /var/www/$PACKAGENAME
+sudo cp /opt/codebender/$PACKAGENAME/apache-config-2.4 /etc/apache2/sites-available/codebender-$PACKAGENAME
+=======
```
Comment (dastergon) [inline scripts/apache_install.sh]: same here, conflict markers

File: scripts/apache_install.sh [hunk 5]
```diff
@@ -4,8 +4,12 @@ set -e
...
 sudo cp /opt/codebender/$PACKAGENAME/apache-config /etc/apache2/sites-available/codebender-$PACKAGENAME
+>>>>>>> 1062d777a61b66db589845a8fb1a2c46aeb6fe56
```
Comment (dastergon) [inline scripts/apache_install.sh]: same here, conflict markers

**Comment by dastergon** [comment]
Any update on the PR? Is it ready for merge?

**Comment by fpapadopou** [comment]
@spiliot @dastergon this should soon be ready. Please have a look at the `Merge Process` section of the  PR message

**Pull request merged**